sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -12,8 +12,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableClientSdkAnalyzers>true</EnableClientSdkAnalyzers>
     <EnableBannedApiAnalyzers>false</EnableBannedApiAnalyzers>
-    <ApiCompatBaselineTargetFramework Condition="'$(TargetFramework)' == 'net462'">net461</ApiCompatBaselineTargetFramework>
-    <ApiCompatBaselineTargetFramework Condition="'$(TargetFramework)' == 'net472'">net461</ApiCompatBaselineTargetFramework>
+    <ApiCompatBaselineTargetFramework Condition="'$(TargetFramework)' == 'net472'">net462</ApiCompatBaselineTargetFramework>
```

Comment:
I leave this to you, but I suspect you can remove this condition as well as I see a net472 binary in the latest 1.46.0 GA package.